ui/components/src/EChart/EChart.tsx

```diff
@@ -105,6 +105,7 @@ export interface EChartsProps<T> {
   sx?: SxProps<Theme>;
   onEvents?: OnEventsType<T>;
   _instance?: React.MutableRefObject<ECharts | undefined>;
+  syncGroup?: string;
```

Comment:
If you're still unsure of if it's a good idea to expose this prop, you can throw on the `__experimental` prefix. However, I think it should be a good abstraction / we can change it if needed anyway.

Reply:
I'm good with exposing it without `__experimental`, this is a pretty popular request

Reply:
To be clear, I'm good with exposing it since it is off by default, embed users will have to opt-in to this by explicitly setting a `syncGroup`--it is only "on by default" for dashboards because of this line in [TimeSeriesChartPanel](https://github.com/perses/perses/blob/e30500c773feaa10cf0229f8a20e35df9c897e90/ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx#L295)

Eventually, I like your idea of having a dashboard-level setting to control this if users want to opt out for all panels